src/annotator/sidebar.js

```diff
@@ -215,19 +215,6 @@ export default class Sidebar {
     });
   }
 
-  destroy() {
```

Comment:
I prefer to have the private methods on top and the public ones a the bottom.

Reply:
This isn't a coding convention we have as a team though. If anything our existing code leans more towards the opposite approach, which has the advantage that the public interface consumers need to know about is near the top. We can discuss this together at some point, but now is not the best time as I think it will be a distraction from functional improvements. Can you leave this method where it is for the moment.